build.gradle

```diff
@@ -67,6 +67,10 @@ subprojects {
 }
 
 configure(subprojects.findAll {it.name != 'data-prepper-api'}) {
+    test {
```

Comment:
We should add this under `subprojects` instead.

Reply:
Fixed. Now it applies to all subprojects.